projects/demo/src/modules/directives/dropdown-context/examples/2/index.ts

```diff
@@ -2,12 +2,34 @@ import {NgForOf} from '@angular/common';
 import {Component, inject} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {
+    TuiTable,
```

Comment:
`TuiTable` is enough, it's already a constant with all other standalone directives:
https://github.com/taiga-family/taiga-ui/blob/main/projects/addon-table/components/table/table.ts

Reply:
Ahh, I see! Thanks for informing me.

Resolved!

Reply:
Resolved!